### DISCUSSION TITLE
Spanish language

### DISCUSSION THREAD

**Discussion by agora-coffee** [LykosAI/StabilityMatrix#185] [General]
I would like to take care of the translation into Spanish (Spain). How can I collaborate?

**Comment by mohnjiles** [comment]
Hi, could you provide an email address that we can add to our POEditor translation project please? Feel free to reach out on our Discord if you don't want to share it here on GitHub. 

Thank you!

**Reply by agora-coffee** [reply]
Sure, cgbaena@gmail.com

**Reply by agora-coffee** [reply]
I think it's time to expand the content to be translated into Spanish. Can you update the poeditor.com list so I can continue translating the new terms?
Thanks

**Reply by ionite34** [reply]
> I think it's time to expand the content to be translated into Spanish. Can you update the poeditor.com list so I can continue translating the new terms? Thanks

We just updated PoeEdit with the current untranslated items, thanks again for working on the translations!

**Reply by agora-coffee** [reply]
Spanish 100% translated. I'm missing more texts that I see in the app but they don't appear in POEditor to translate. Is this normal?

**Reply by agora-coffee** [reply]
For example: in the Launch settings menu (Launch Options), nothing is translated.

**Reply by agora-coffee** [reply]
In Settings > Inference menu, nothing is translated

**Reply by agora-coffee** [reply]
In the Inference app aren't translated:
Top line: Config, Prompt, Image Output and Gallery.
Lateral left menu: Sampler, Scheduler, FreeU, Hires Fix, Upscale, Seed, Batch Size and Batches.
Center: Prompt, Negative Prompt and Generate Image

**Reply by agora-coffee** [reply]
Principal screen:
Launch, Inference, Packages, Checkpoints, Model Browser, Output Browser and Settings are not translated
